client/src/api/schema/schema.ts

```diff
@@ -18968,32 +18972,38 @@ export interface components {
         Requirement: "logged_in" | "new_history" | "admin";
         /** ResourceRequirement */
         ResourceRequirement: {
-            /** Cores Max */
-            cores_max: number | null;
-            /** Cores Min */
+            /** cores_max */
+            cores_max?: number | null;
```

Comment:
Should the description be shown here?
Same for other resource requirements.

Reply:
That's an `openapi-typescript` thing, it is in the full openapi schema. For the suggestions we care about `ToolSourceSchema.json` and it is in there.

Reply:
sure